src/jfem.jl

```diff
@@ -61,13 +59,15 @@ export dx, ds,dS,dP
 
 
 @fenicsclass Matrix
-assemble(assembly_item::Form)=Matrix(fenics.assemble(assembly_item.pyobject)) #this gives as PETScMatrix of appopriate dimensions
+#assemble(assembly_item::Form)=Matrix(fenics.assemble(assembly_item.pyobject))
+assemble(assembly_item::Form;tensor=nothing, form_compiler_parameters=nothing, add_values=false, finalize_tensor=true, keep_diagonal=false, backend=nothing) = Matrix(
+fenics.assemble(assembly_item.pyobject,tensor=tensor,form_compiler_parameters=form_compiler_parameters,add_values=add_values,finalize_tensor=finalize_tensor,keep_diagonal=keep_diagonal,backend=backend))#this gives as PETScMatrix of appopriate dimensions
```

Comment:
is a PETScMatrix a type we have? Should we depend on PETSc.jl and wrap this somehow?